src/color-variants.css

```diff
@@ -2143,35 +2143,35 @@ input:checked + .switch--dot-transparent::after {
  * <div class='shadow-darken25'>shadow-darken25</div>
  */
 .shadow-darken10 {
-  box-shadow: 0 0 10px 2px var(--darken10) !important;
+  box-shadow: 0 2px 10px 0px var(--darken10) !important;
```

Comment:
🤔  weird that a unit for `0` is being appended

Reply:
Oh @tristen I forgot to re-build. Try now